dactyl-config.yml

```diff
@@ -763,6 +763,14 @@ pages:
         targets:
             - local
 
+    -   md: use-cases/run-a-rippled-validator.md
+        html: run-a-rippled-validator.html
+        funnel: Use Cases
+        template: template-use-case.html
+        # jha TODO - add useful_background
```

Comment:
suggested background / additional resources:

- consensus principles & rules
- the rippled server
- amendments
- fee voting